test/api/v1/site_controller_test.exs

```diff
@@ -75,8 +75,8 @@ defmodule SituationRoom.Site.ControllerTest do
       conn = conn(:get, "/sites/345324562")
       conn = Router.call(conn, @opts)
       assert conn.state == :sent
-      assert conn.status == 400
-      assert conn.resp_body == "{'error': 'Site 345324562 was not be found..'}"
+      assert conn.status == 404
+      assert conn.resp_body == "Not Found"
```

Comment:
Don't need to change this, just adding it as a side note, but in this case also asserting on the `resp_body` really doesn't add anything to this test.  Checking that the status was a 404 is enough, as if/when that is the case we typically don't care at all what is in the response body - and in many cases the body will be empty completely.  In our example, we know it has content because we control the code in the controller and wrote the response handler,  but in general that wont' always be the case.

In your tests, you want to test as much/as little as is required to prove out the behaviour you are looking for.  Checking the boddy here above and beyond the status == 404 doesn't add anything of use to this test.